x-pack/plugins/ml/public/embeddables/anomaly_charts/anomaly_charts_initializer.tsx

```diff
@@ -112,9 +146,9 @@ export const AnomalyChartsInitializer: FC<AnomalyChartsInitializerProps> = ({
             />
           </EuiFormRow>
         </EuiForm>
-      </EuiModalBody>
+      </EuiFlyoutBody>
 
-      <EuiModalFooter>
+      <EuiFlyoutFooter>
```

Comment:
The footer content isn't positioned at the bottom of the flyout. Also for consistency with our other flyouts, I think we should use 'Confirm' rather than 'Confirm configurations' for the 'apply' button text:

<img width="695" alt="Screenshot 2024-05-20 at 10 13 52" src="https://github.com/elastic/kibana/assets/7405507/ffd390eb-f6db-4e48-ae51-450c35ff48bf">

Anomaly swim lane buttons:

<img width="671" alt="Screenshot 2024-05-20 at 10 18 47" src="https://github.com/elastic/kibana/assets/7405507/9460edc6-ab35-48b0-b90a-0884258582c1">

Reply:
Vertical alignment of the footer is looking good, but shouldn't the 'Confirm' button be right-aligned in the footer?

<img width="706" alt="Screenshot 2024-05-21 at 12 03 23" src="https://github.com/elastic/kibana/assets/7405507/513446a2-4924-412d-8864-1e7d6bba3822">

e.g. Single Metric Viewer:

<img width="636" alt="Screenshot 2024-05-21 at 12 05 21" src="https://github.com/elastic/kibana/assets/7405507/d4ac8f2f-042e-4a32-bb5a-f1b8276be732">

Reply:
Fixed [here](https://github.com/elastic/kibana/pull/183456/commits/4a99e54a475542caf16f4a7b7701f1f2e2a89242)